blog/migrations/0035_alter_blogpage_lead_image.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 3.2.19 on 2023-06-02 06:51
+
+import common.models.helpers
+from django.db import migrations
+import wagtail.blocks
+import wagtail.fields
+import wagtail.images.blocks
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('blog', '0034_update_and_migrate_incident_set_categories'),
+    ]
+
+    operations = [
+        migrations.RemoveField(model_name='blogpage', name='lead_image'),
```

Comment:
TODO: I don't think this is quite right because it will delete the old image when instead we should be moving it into the new schema.

Reply:
Feel free to ping @chigby about this today if you need any help with the necessary data migration.